### DISCUSSION TITLE
Row authorization in collection.

### DISCUSSION THREAD

**Discussion by tohodakilla** [pocketbase/pocketbase#918] [Q&A] [answered]
Hi!

First of all I would like to thank you from the bottom of my heart for this incredible project! ❤️❤️❤️

Im making an application using pocketbase v0.8.0-rc2
I have created auth collection called _tenants_ and _products_ collection.
![Screenshot from 2022-11-03 16-11-34](https://user-images.githubusercontent.com/1858683/199760923-c4d609b8-dfa1-4e6f-b6a6-a6df5fce88ab.png)

![Screenshot from 2022-11-03 16-11-53](https://user-images.githubusercontent.com/1858683/199761000-9d65387d-a358-481b-8f18-3f2ecf363e8b.png)

Products has a relational field called tenant that points to tenant who has created the record.
Im trying to only allow tenants who created records to view them.

I have put following condition: @request.auth.id = tenant.id
![Screenshot from 2022-11-03 16-12-06](https://user-images.githubusercontent.com/1858683/199761035-21eb5ef3-9834-4bc7-927e-9f424cf811dc.png)

But when i make request authenticated with proper tenant it returns empty list, if i remove this condition it shows product.
Attached screenshots of current pocketbase connections state & configutration.

Please assist.
❤️🙏

**Comment by ganigeorgiev** [comment, marked as answer]
@tohodakilla I'm not able to reproduce the issue. I've tested both locally and with the live demo (https://pocketbase.io/_/).

How you are sending the request? Make sure that the `Authorization` header is properly formatted (in v0.8.0 the value must be only the token, aka. `Authorization: YOUR_TOKEN`).

**Reply by tohodakilla** [reply]
I have restarted server and changed request header to: `Authorization: YOUR_TOKEN` instead of `Authorization: User YOUR_TOKEN`

and it worked just fine, sorry to bother you with such miniscule thing.

:heart: :pray:

**Reply by ganigeorgiev** [reply]
@tohodakilla Yes, with v0.8.x the schema is embedded in the token as payload data and is no longer needed. The documentation will be updated very soon with the stable v0.8.0 release.
